src/content/reference/react/useRef.md

```diff
@@ -228,11 +228,7 @@ function MyComponent() {
 
 Si *tienes* que leer [o escribir](/reference/react/useState#storing-information-from-previous-renders) algo durante el renderizado, [utiliza el estado](/reference/react/useState) en su lugar.
 
-<<<<<<< HEAD
 Si rompes estas reglas, tu componente puede seguir funcionando, pero la mayoría de las nuevas características que estamos añadiendo a React se basarán en estas expectativas. Lee más sobre [mantener tus componentes puros.](/learn/keeping-components-pure#where-you-can-cause-side-effects)
```

Comment:
@mateoguzmana Creo que hubo un problema al resolver este conflicto. El enlace correcto es `/learn/keeping-components-pure#where-you-_can_-cause-side-effect` no `/learn/keeping-components-pure#where-you-can-cause-side-effects`

Reply:
Es cierto, acabo de probar y estaba mal. Arreglado: https://github.com/reactjs/es.react.dev/pull/774 @carburo

Reply:
Ah, ya vi. El enlace original que estaba en main, ya estaba corrupto 😇